packages/stitch/tests/stitchSchemasPathBug.test.ts

```diff
@@ -0,0 +1,96 @@
+import { describe, expect, test } from 'vitest';
+import { makeExecutableSchema } from '@graphql-tools/schema';
+import { stitchSchemas } from '@graphql-tools/stitch';
+import { graphql, OperationTypeNode } from 'graphql';
+import { delegateToSchema } from '@graphql-tools/delegate';
+
+describe('Supergraph handles stitched field with different name correctly', () => {
+    test('ensures correct error path in response', async () => {
+        const subschema = makeExecutableSchema({
+            typeDefs: /* GraphQL */ `
+            type Query {
+                books(ids: [ID!]!): [Book!]!
+            }
+
+            type Book {
+                id: ID!
+                title: String
+                summary: String
+            }
+            `,
+            resolvers: {
+                Query: {
+                    books: (_parent, args) => ([{
+                        id: args.id,
+                        title: () => {
+                            throw new Error('Title - Forced error for testing purposes');
+                        },
+                        summary: () => {
+                            throw new Error('Summary - Forced error for testing purposes');
+                        },
+                    }]),
+                },
+            },
+        });
+
+        // Stitch the subschema into the supergraph
+        const supergraphSchema = stitchSchemas({
+            subschemas: [subschema],
+            typeDefs: /* GraphQL */ `
+                type Query {
+                    book(id: ID!): Book
+                }
+
+                type Book {
+                    id: ID!
+                    title: String
+                    summary: String
+                }
+            `,
+            resolvers: {
+                Query: {
+                    book: {
+                        selectionSet: '{ id }',
```

Comment:
Also I think this is incorrect. `selectionSet` is used to get values from the root. it is `Query` in this case;
https://the-guild.dev/graphql/stitching/docs/approaches/schema-extensions#selectionset

Reply:
True, this could be omitted. I don't need `selectionSet` at all here